src/Iot.Device.Bindings/Iot.Device.Bindings.csproj

```diff
@@ -18,6 +19,8 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.Drawing.Common" Version="4.6.0-preview4.19164.7" />
+
```

Comment:
nit: the line below - I know this isn't you fully related but since you're visiting the file in the line below you can remove `$(MSBuildThisFileDirectory)`

Reply:
done!